tests/integration/test_charms.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from pathlib import Path
```

Comment:
I suggest you check best practices for python imports so we can improve this import section.